The Archean origin of assimilatory sulfate metabolisms provides novel insight into redox conditions of early Earth environments

Dissimilatory sulfur metabolisms recording differing biological isotopic fractionation are well studied, important components of sulfur cycling (Mateos et al., 2023). Assimilatory sulfur metabolisms and genes across life provide a complementary window into sulfur biogeochemistry with individual pathways having specific isotopic fractionations acting on distinct redox states (e.g. sulfate, sulfide, sulfite) for anabolism (Liu et al., 2012). An assimilation pathway exists, which starts with sulfate adenylyltransferase (sat/ATP sulfurylase) catalyzing a reaction of adenosine triphosphate (ATP) and sulfate (SO42-) resulting in adenosine 5’-phosphosulfate (APS), and incorporation of more reduced sulfur into biomolecules. This sat/ATP sulfurylase enzyme represents the first step required by life to incorporate sulfur and informs our understanding of biological processes performing this fundamental chemical reaction. A phylogenetic and molecular clock analysis of the sat/ATP sulfurylase protein family (E.C. 2.7.7.4) was performed to determine the age of sulfate assimilation proteins. Extant diversity of sat proteins was estimated to have a last common ancestor ~3.24 Ga (95% CI 3.51–3.06 Ga) using relaxed molecular clocks with uncorrelated rates, calibrated with eukaryotic and cyanobacteria age ranges from previously determined fossil calibrated dates. These results suggest sulfate cycling in Paleoarchean environments, despite extensive evidence of low marine sulfate concentrations (Crowe & Canfield et al., 2014). Archean sulfate biogeochemical cycling could result from microbial sulfur oxidation and sources could include abiotic oxidation of volcanic sulfur, hydrothermal processes or pyrite (Canfield, 2001, Lyons et al., 2024). This phylogenomic evidence of sulfate during Archean times provides an independent complement to geochemical records and allows insight into environmental niches and redox potentials.

Clock Calibration Scheme	Root ONLY
(Prior 4.3-3.0 Ga) Posterior ages (1)	Root + Eukarya ONLY
Posterior ages (2)	Combined: Root + Eukarya + Cyanobacteria (ALL)
Posterior ages (4)	Root + Cyanobacteria ONLY
Posterior ages (3)
Branch Rate Model				
UGAM	3584 (4288 3030)	3063 (3330 3001)	3240 (3518 3064)	3871 (4295 3318)
CIR	3615 (4256 3045)	3076 (3263 3002)	3557 (4060 3297)	3934 (4299 3489)
![image](https://github.com/user-attachments/assets/6acb8150-2022-4cd7-b782-f75016b48642)

Table 1. Molecular clock results for crown sat/ATPS/met3 protein family age across different calibration sets (1-4) and using uncorrelated (UGAM – Uncorrelated Gamma distribution) and autocorrelated (CIR – Cox Ingersoll Ross) branch rate models.

<img width="468" alt="image" src="https://github.com/user-attachments/assets/8600f88d-ecda-4cf0-a52e-2a510b55a4da">

Figure 1. Crown sat/ATPS protein family divergence date age estimates (probability densities). Blue: based on molecular clock model estimates under Cyanobacteria only calibrations. Red: based on molecular clock model estimates under Eukaryotic only calibrations. Purple: based on molecular clock model estimates under all calibrations. Branch rate models: UGAM (uncorrelated gamma distribution) in solid line, CIR (Cox Ingersoll Ross model) in dotted lines for uniform tree process prior (default). Data from 100 chronograms taken after convergence and plotted as Bayesian posterior probabilities (Y) with age in millions of years ago (X), with a kernel density estimate (KDE), formally a Cumulative Density Function, using the python package seaborn.

<img width="140" alt="image" src="https://github.com/user-attachments/assets/52dabd43-8abb-48ea-a74e-21ba83c05fe7">

Figure 2A. Bayesian Chronogram (Phylobayes) run under uniform tree process prior LG substitution model and CIR branch rate model.

<img width="245" alt="image" src="https://github.com/user-attachments/assets/50b2c8fc-3992-4405-8901-40763bac6b5d">

Figure 2B. Maximum Likelihood (IQ-Tree) phylogenetic tree run under LG+R10 model, Minimal Ancestor Deviation (MAD) Rooted with branch lengths scale in substitutions per site.

<img width="213" alt="image" src="https://github.com/user-attachments/assets/22b16a14-2de9-4b79-9e1e-688514d928f4">

Figure 2C. Maximum Likelihood (IQ-Tree) phylogenetic tree run under LG+R10 model with bootstraps supports on each bipartition (sh-alrt/UFBoot).
